### ISSUE TITLE
Use a math library

### ISSUE THREAD

**Issue by szepeviktor** [PrinsFrank/sunsetrise#4]
For example https://github.com/brick/math

PHP's floating point calculations will surely drive you crazy 😵

**Comment by PrinsFrank** [comment]
Yeah, I probably will when I have a working prototype. Right now I don't even get valid results. ;)
